### PR TITLE
fix(huey): Zipper length calculation, clarify it is a maximum

### PR DESCRIPTION
**Pull request by joostdecock**
This stems from the excellent work by @HaasJona in #6421 in which they requested to double-check the zipper length calculation. Which I didn't get around to doing in the code review, but was able to now.

Turns out there was a slight issue with it, so this resolves it. I've also updated the wording to clarify that what we are calculating as the __maximum__ zipper length. Since zippers typically come is standard lengths, it's typically not possible to find the exact length you need, so it's IMHO important to make it clear to the user they should always purchase a zipper that is as close as possible, yet shorter than the calculated maximum length.

### CODE REVIEW

File: config/changelog.yaml [hunk 1]
```diff
@@ -22,6 +22,8 @@ Unreleased:
   Fixed:
     huey:
       - Fixed an issue where ribbing height would distort the pocket under certain conditions. Fixes #6238
+      - Adding missing translation for rubbing option
```
Comment (HaasJona) [inline config/changelog.yaml:25]: ribbing not rubbing?

**Review by HaasJona — COMMENTED** [review]
Looks good except for the typo. Didn't check the length calculation, but since you corrected mine, I assume you got it correct.

File: config/changelog.yaml (hunk 1 above)
Comment (joostdecock) [reply config/changelog.yaml:25]: That's what she said :joy:

Comment (joostdecock) [reply config/changelog.yaml:25]: (I've fixed it)

**Pull request merged**